src/runtime/virtcontainers/sandbox.go

```diff
@@ -448,13 +448,20 @@ func createAssets(ctx context.Context, sandboxConfig *SandboxConfig) error {
 	defer span.End()
 
 	for _, name := range types.AssetTypes() {
-		a, err := types.NewAsset(sandboxConfig.Annotations, name)
+		annotation, _, err := name.Annotations()
 		if err != nil {
 			return err
 		}
+		// For remote hypervisor donot check for Absolute Path incase of ImagePath, as it denotes the name of the image.
```

Comment:
So that here we could have something like:
```
if sandboxConfig.HypervisorType != RemoteHypervisor && annotation != sandboxConfig.HypervisorConfig.ImagePath {
    if value != "" {
			  config.HypervisorConfig.ImagePath = value
		}
} else {
   <old workflow>?
}
```